pyanaconda/ui/gui/hubs/__init__.py

```diff
@@ -172,7 +172,7 @@ def _createBox(self):
                     spoke.initialize()
                     continue
 
-                spoke.selector = AnacondaWidgets.SpokeSelector(C_("GUI|Spoke", spoke.title),
```

Comment:
Shouldn't you use `C_("GUI|Spoke", "...")` for all titles of our spokes?

Reply:
Yep, definitely! Fixing it now. :)